pkg/dinghyfile/builder.go

```diff
@@ -18,13 +18,16 @@ package dinghyfile
 
 import (
 	"bytes"
+	"encoding/json"
 	"errors"
+	"fmt"
+	"time"
```

Comment:
is this import list supposed to be in alphabetical?

Reply:
Not sure, it's what `go-fmt` decided to do to my imports so I let it do it's thing!

Reply:
It's best practice to be alphabetical because of the way Go importing mechanism works, so that import order is predictable. That's why tools like `go-fmt` automatically do that.